backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/LlvmDeclarations.kt

```diff
@@ -346,7 +345,7 @@ private class DeclarationsGeneratorVisitor(override val context: Context) :
             if (declaration.isTypedIntrinsic || declaration.isObjCBridgeBased()
                     // All call-sites to external accessors to interop properties
                     // are lowered by InteropLowering.
-                    || (declaration.isAccessor && declaration.isFromMetadataInteropLibrary())
+                    || (declaration.isAccessor && declaration.isFromInteropLibrary())
```

Comment:
It is not clear if it can actually be `true` here.
If this declaration is from metadata interop library, then why do we visit it here?

Reply:
(on the other hand, the new `isFromInteropLibrary` function is anyway required in other place, so there are no more reasons to figure this code out in this PR).